[docs] Grammar fix in FAQ (#14974)
mui-org/material-ui: docs/src/pages/getting-started/faq/faq.md

- You accidentally **bundle** two versions of Material-UI. You might have a dependency not correctly setting Material-UI as a peer dependency.
- You are using `StylesProvider` for a **subset** of your React Tree.

> - You are using a bundler and it is splitting code in a way that results in multiple class name generator instances to be created.

Reviewer: ```suggestion
- You are using a bundler and it is splitting code in a way that results in multiple class name generator instances being created.
```
```suggestion
- You are using a bundler and it is splitting code in a way that causes multiple class name generator instances to be created.
```

(Pick one...)

> > If you are using webpack with the [SplitChunksPlugin](https://webpack.js.org/plugins/split-chunks-plugin/), try configuring the [`runtimeChunk` setting under `optimizations`](https://webpack.js.org/configuration/optimization/#optimization-runtimechunk).

Reviewer: ```suggestion

> If you are using webpack with the [SplitChunksPlugin](https://webpack.js.org/plugins/split-chunks-plugin/), try configuring the [`runtimeChunk` setting under `optimizations`](https://webpack.js.org/configuration/optimization/#optimization-runtimechunk).
```
